index.js

```diff
@@ -206,6 +213,17 @@ Shopify.prototype.graphql = function graphql(data) {
       this.updateGqlLimits(res.body.extensions.cost.throttleStatus);
     }
 
+    if (res.body.errors) {
+      const first = res.body.errors[0];
+      const err = new Error(first.message);
```

Comment:
With the REST API this is one of https://github.com/sindresorhus/got/tree/v8.3.1#errors but I don't any of those classes makes sense here. I think a generic `Error` is ok. What do you think?

Reply:
Yeah, generic error is simplest for now I think.